Parsers/ASimFileEvent/Parsers/imFileEvent.yaml

```diff
@@ -64,6 +64,7 @@ Parsers:
   - _Im_FileEvent_MicrosoftWindowsEvent
   - _Im_FileEvent_Native
   - _Im_FileEvent_SentinelOne
+  - _Im_FileEvent_VMwareCarbonBlackCloud
 ParserQuery: |
   let DisabledParsers=materialize(_GetWatchlist('ASimDisabledParsers')
   | where SearchKey in ('Any', 'ExcludevimFile')
```

Comment:
@jayeshprajapaticrest Please include VMWareCarbon caller in ParserQuery property

Reply:
Done